src/apps/loyalty/server/index.tsx

```diff
@@ -0,0 +1,46 @@
+import * as express from "express"
+import * as path from "path"
+import * as React from "react"
+import { renderToString } from "react-dom/server"
+import * as styleSheet from "styled-components/lib/models/StyleSheet"
+import renderPage from "./template"
+
+import Inquiries from "../containers/inquiries"
+import Login from "../containers/login"
+
+const app = express()
+
+app.use(express.static(path.resolve(__dirname)))
+
+app.get("/", (req, res) => {
+  // check if the user is logged in, if not redirect to /login
+  // else render /inquiries
+  res.redirect(req.baseUrl + "/inquiries")
+})
+
+app.get("/login", (req, res) => {
+  const html = renderToString(<Login />)
+  const styles = styleSheet.rules().map(rule => rule.cssText).join("\n")
+  res.send(renderPage({
+    styles,
+    html,
+    entrypoint: "/bundles/login.js",
+  }))
+})
+
+app.get("/inquiries", (req, res) => {
+  const user = (req as any).user
```

Comment:
oh i see, here's the check.

This seems to assume that you're running this 'inside' of Force, but in dev what's the workflow? It seems like now we _have_ to mount this inside of Force, have both Force and reaction-force running, and then work on this.

It feels really weird to be referring to methods defined in other repos...what if I don't have Force running? I feel like this PR (or the next one) can use some sort of README or documentation about the moving parts and the intended flow.